rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/520_fetch_fields.yml

```diff
@@ -128,18 +128,76 @@ fetch _seq_no via stored_fields:
 
 ---
 fetch _seq_no via fields:
+  - requires:
+      cluster_features: ["error_code_changed"]
```

Comment:
I added a `NodeFeature` in `MapperFeatures:: getTestFeatures` to flag this method for execution.  The naming is generic and can be used in other cases if needed.